module/type/TypeBrowser.kt

```diff
@@ -98,12 +98,12 @@ class TypeBrowser(isOpen: Boolean = false, order: Int) : Browsers.Browser(isOpen
     private fun exportButton(navState: Navigator.NavigatorState<ThingTypeState<*, *>>) = IconButtonArg(
         icon = Icon.EXPORT,
         enabled = Service.project.current != null && !Service.schema.hasRunningCommand,
-        tooltip = Tooltip.Arg(title = Label.EXPORT_SCHEMA)
+        tooltip = Tooltip.Arg(title = Label.EXPORT_TYPE_SCHEMA)
```

Comment:
ahh ok i think actually `EXPORT_SCHEMA_TYPES` and then reflecting this naming in the methods and variables below makes the most sense!
this would also make your constants match the reverse naming convention we using for scoping:
```
EXPORT_SCHEMA
EXPORT_SCHEMA_TYPES```
as opposed to 
```
EXPORT_SCHEMA
EXPORT_TYPE_SCHEMA
```

Reply:
I think this would be good, but might require a deeper change: we define the methods for getting the schema types in client-java as `Database.typeSchema()` and `Database.ruleSchema()`. Do we want to change these functions to `schemaTypes` and `schemaRules` respectively?